### DISCUSSION TITLE
Authenticatie dashboard

### DISCUSSION THREAD

**Discussion by rkops-bd** [dsmrreader/dsmr-reader#1473] [Questions & answers / How-to's] [answered]
Het zou fijn zijn om het dashboard te voorzien van (optionele) authenticatie.
Als ik de DMSR wil exposen op het internet, wil ik daar wel graag een vorm van authenticatie voor hebben.

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag. Er zit in de front-end settings een checkbox waarmee je de hele applicatie achter login krijgt. Laat maar weten als je hem niet kan vinden.

**Comment by dennissiemensma** [comment]
Je kunt het instellen met de checkbox bovenin: `/admin/dsmr_frontend/frontendsettings/1/change/`

**Comment by rkops-bd** [comment]
Ik heb het inderdaad gevonden. Dank voor de tip!
